src/jit/codegenarmarch.cpp

```diff
@@ -2103,16 +2103,25 @@ void CodeGen::genCallInstruction(GenTreeCall* call)
         void* addr = nullptr;
         if (callType == CT_HELPER)
         {
-            // Direct call to a helper method.
-            CorInfoHelpFunc helperNum = compiler->eeGetHelperNum(methHnd);
-            noway_assert(helperNum != CORINFO_HELP_UNDEF);
+// Direct call to a helper method.
+#ifdef FEATURE_READYTORUN_COMPILER
+            if (call->gtEntryPoint.addr != NULL)
+            {
+                addr = call->gtEntryPoint.addr;
```

Comment:
Does this need to have the case of `accessType == IAT_PVALUE`? 

If not, it would be useful to add assert it at least.

Reply:
@jkotas I don't see any using of accessType now. May be later when this code will be developed more deeply